src/topology/algebra/module.lean

```diff
@@ -112,6 +112,27 @@ lemma is_open_map_smul_of_unit (a : units R) : is_open_map (λ (x : M), (a : R)
 lemma is_closed_map_smul_of_unit (a : units R) : is_closed_map (λ (x : M), (a : R) • x) :=
 (homeomorph.smul_of_unit a).is_closed_map
 
+/-- If `M` is a topological module over `R` and `0` is not an isolated point of `R`,
```

Comment:
The docstring does not correspond to the statement: what matters is not whether `0` is isolated, but whether it is a limit of invertible elements.

Reply:
Fixed.